src/storage/hummock_sdk/src/sstable_info.rs

```diff
@@ -118,6 +125,11 @@ impl From<&PbSstableInfo> for SstableInfo {
             range_tombstone_count: pb_sstable_info.range_tombstone_count,
             bloom_filter_kind: PbBloomFilterType::try_from(pb_sstable_info.bloom_filter_kind)
                 .unwrap(),
+            sst_size: if pb_sstable_info.sst_size == 0 {
+                pb_sstable_info.file_size
+            } else {
+                pb_sstable_info.sst_size
+            },
```

Comment:
Can we ensure that `sst_size` will never be zero for newly written sst? I am thinking whether it is safe to use 0 to determine whether this field is filled. Another alternative is to make `sst_size` optional in pb so that we can check more accurately.